### DISCUSSION TITLE
Consider simplifying `path_id` matcher to match anything following the forward slash

### DISCUSSION THREAD

**Discussion by adam12** [jeremyevans/autoforme#45] [General]
I have some models that use text primary key which contains a reference to a Ruby constant. This will contain `::`, which the current regular expression (in the Roda framework handler at least, but the others look similar) will not set as the path id.

Instead, the value is set in the URL as the `id` query string param, which will appear to work until the form is submitted. When rendering the form, the path in the `[action]` attribute looks correct, but the `id` is ignored, causing an exception when Sequel cannot find the record of `nil`.

The current expression is matching alphanumeric + `-` but I wonder if we can just relax it to `.+`, and if there are any drawbacks.

Here is the matcher in question: https://github.com/jeremyevans/autoforme/blob/149ffc7f9d412c1432f833cc1514ef640e04816d/lib/autoforme/frameworks/roda.rb#L26

**Comment by jeremyevans** [comment]
Not sure if we want to switch to `.+` (that even allows `/`), but adding `:` should be fine.  Do you think you could submit a pull request for that?  I would also be OK making this configurable.

**Reply by adam12** [reply]
I'll submit a PR for adding the `:` to the regular expression. If I can make it configurable without a huge lift, I'll explore that as well. Thanks.
